web/src/app/components/job-list-item/job-list-item.component.ts

```diff
@@ -18,7 +18,7 @@ import {Component, Input, OnInit, OnDestroy} from '@angular/core';
 import {DialogService} from 'app/services/dialog/dialog.service';
 import {ImportDialogComponent} from 'app/components/import-dialog/import-dialog.component';
 import {Job} from 'app/models/job.model';
-import {getPinImageSource} from 'app/components/map/ground-pin';
+import {getPinImageSource} from 'app/ground-pin';
```

Comment:
We need to find home for `ground-pin`, it can just sit right under `app/`, maybe `assets/` (it's not a image file, but it's providing a svg dynamically)? Or somewhere else?

Reply:
Files in `assets` are actually where static images get served from. Maybe a service is more appropriate? Trying this now..